src/App.js

```diff
@@ -1,6 +1,10 @@
+import Calculator from './component/calculator';
+
 function App() {
   return (
-    <div className="App" />
+    <div className="App">
+      <Calculator />
+    </div>
   );
 }
```

Comment:
Great job @shella12 🤝

- [ ] There seems to be an error with the way Calculator is imported. I think you need to export Calculator in `calculator.js`.
![error](https://user-images.githubusercontent.com/63917725/204382115-6cf72a66-99d1-427b-9b32-9e70963df522.PNG)